BranchInfo.props

```diff
@@ -3,8 +3,10 @@
     <MajorVersion>2</MajorVersion>
     <MinorVersion>1</MinorVersion>
     <PatchVersion>0</PatchVersion>
+    <StabilizePackageVersion Condition="'$(ProduceStablePackages)' != ''">$(ProduceStablePackages)</StabilizePackageVersion>
     <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">false</StabilizePackageVersion>
```

Comment:
Does it make sense to keep the `StabilizePackageVersion` version around?  What if we just started using `ProduceStablePackages` everywhere?

Reply:
+1

Reply:
While we could use the same property it causes some weird issues once it is passed as a global msbuild property that cannot be changed. For a boolean that will either be true or false that is less of an issue. I will change this to be the same property name.